ror/spec/workers/comment_status_check_worker_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe CommentStatusCheckWorker, type: :worker do
+  describe '.perform' do
```

Comment:
```suggestion
  describe '#perform' do
```

`#` denotes instance method, `.` denotes class method

Reply:
Fixed.